### ISSUE TITLE
avatar related CSP violations in stage (report-only) 

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa-content-server#1724]
See https://github.com/mozilla-services/puppet-config/issues/623

In stage, this header is added:

```
  Content-Security-Policy-Report-Only:"default-src 'self' https://api-accounts.stage.mozaws.net https://oauth.stage.mozaws.net;\
      img-src 'self' data:; report-uri https://fxa-csp-violations.stage.mozaws.net/_/csp-violation"
```

and reports are coming back since profile.stage.mozaws.net is not included. e.g., 

```
{
    "csp-report": {
	"document-uri":"https://accounts.stage.mozaws.net/settings/avatar",
	"referrer":"",
	"blocked-uri":"https://profile.stage.mozaws.net/v1/avatar",
	"violated-directive":"default-src https://accounts.stage.mozaws.net:443 https://api-accounts.stage.mozaws.net:443 https://oauth.stage.mozaws.net:443"
    }
}
```

**Comment by issues-bouncer** [comment]
Hey @jrgm, you have been assigned on this issue for more than 14 days!
Maybe you should consider un-assigning yourself and let other people try to solve this issue.

**Comment by jrgm** [comment]
This should have been closed with #2278. Closing.
